doc/source/tutorial/optimize.rst

```diff
@@ -916,6 +915,169 @@ We'll now plot all found minima on a heatmap of the function::
    :alt: "This X-Y plot is a heatmap with the Z value denoted with the lowest points as black and the highest values as white. The image resembles a chess board rotated 45 degrees but heavily smoothed. A red dot is located at many of the minima on the grid resulting from the SHGO optimizer. SHGO shows the global minima as a red X in the top right. A local minima found with dual annealing is a white circle marker in the top left. A different local minima found with basinhopping is a yellow marker in the top center. The code is plotting the differential evolution result as a cyan circle, but it is not visible on the plot. At a glance it's not clear which of these valleys is the true global minima."
    :include-source: 0
 
+Global Optimization Solvers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Global optimization solvers aim to find the global minimum over the entire search space,
+which is especially important when the objective function has multiple local minima.
+
+**Global Optimization Solvers Table:**
+
+.. list-table::
+   :widths: 20 15 15 20 20 15
+   :header-rows: 1
+
+   * - Solver
+     - Uses Gradient
+     - Uses Hessian
+     - Bounds Constraints
+     - Nonlinear Constraints
+     - Utilizes Sparsity
+   * - dual_annealing
+     - (✓)
+     - (✓)
+     - ✓
+     - 
+     - 
+   * - differential_evolution
+     - 
+     - 
+     - ✓
+     - ✓
+     - 
+   * - shgo
+     - (✓)
+     - (✓)
+     - ✓
+     - ✓
+     - 
+   * - direct
+     - 
+     - 
+     - ✓
+     - 
+     - 
+   * - basinhopping
+     - (✓)
+     - (✓)
+     - 
+     - 
+     - 
+
+(✓) = Depending on the chosen local minimizer
+
+
+
+Local optimization
+------------------
+
+Local Optimization Solvers
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Local optimization solvers are algorithms that search for a local minimum, 
+starting from an initial point. These solvers are particularly useful 
+when the problem is presumed to have a smooth landscape around the initial guess.
+
+**Local Optimization Solvers Table:**
+
+.. list-table::
+   :widths: 15 15 15 20 20 15
+   :header-rows: 1
+
+   * - Solver
+     - Uses Gradient
+     - Uses Hessian
```

Comment:
This table could be fleshed out more w.r.t gradient/hessian information. e.g. LBFGSB uses a gradient, but that gradient can be estimated by the numerical differentiation, OR the user can supply a gradient function.
In comparison Newton-CG definitely requires a gradient function, but can estimate hessian by numerical differentiation or hessian function.
Yet again trust-constr can estimate both gradient and hessian by functions, or by numdiff for gradient and HessianUpdateStrategy for the Hessian.

i.e. there is lots of complexity in what each function can use.

Reply:
What about a second table for details about derivatives and hessians?

Reply:
> What about a second table for details about derivatives and hessians?

@andyfaff